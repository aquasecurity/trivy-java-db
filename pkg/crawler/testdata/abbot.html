<!DOCTYPE html>
<html><head>
<meta http-equiv="content-type" content="text/html; charset=windows-1252">
	<title>Central Repository: abbot</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<style>
body {
	background: #fff;
}
	</style>
</head>

<body>
	<header>
		<h1>abbot</h1>
	</header>
	<hr>
	<main>
		<pre id="contents">
<a href="../">../</a>
<a href="abbot/" title="abbot/">abbot/</a>                                                           -         -
		</pre>
	</main>
	<hr>


</body></html>