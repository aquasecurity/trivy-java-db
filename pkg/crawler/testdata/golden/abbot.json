{
  "GroupID": "abbot",
  "ArtifactID": "abbot",
  "Versions": [
    {
      "Version": "0.12.3",
      "SHA1": "UdKKJ9kZzoaQpA9PM1udWRzrFuk=",
      "License": ""
    },
    {
      "Version": "0.13.0",
      "SHA1": "WW2R5nYxsN6wX7aF2NG2c18+T2A=",
      "License": ""
    },
    {
      "Version": "1.4.0",
      "SHA1": "ojY2RqndBZVWM7RQAQtZohr4pCM=",
      "License": "1028932157|2166136261|2425343180|3188410119"
    }
  ],
  "ArchiveType": "jar"
}